internal/chrome_android.py

```diff
@@ -34,14 +34,19 @@
     '--disable-external-intent-requests',
     '--enable-remote-debugging',
     '--net-log-capture-mode=IncludeSensitive',
-    '--load-media-router-component-extension=0',
     '--mute-audio',
     '--disable-hang-monitor',
     '--password-store=basic',
     '--disable-breakpad',
-    '--dont-require-litepage-redirect-infobar',
-    '--override-https-image-compression-infobar',
-    '--disable-fetching-hints-at-navigation-start'
+    '--disable-fetching-hints-at-navigation-start',
```

Comment:
I _think_ `--disable-features=OptimizationHints` may preclude this one. Looks kinda like it but I haven't verified in details.  No big deal either way.